src/ui/popup.js

```diff
@@ -259,6 +293,14 @@ export default class Popup extends Evented {
 
     /**
      * Returns the `Popup`'s HTML element.
+     * @example
+     * // Change the `Popup` element's font size
+     * var popup = new mapboxgl.Popup()
+     *   .setLngLat([-96, 37.8])
+     *   .setHTML("<p>Hello World!</p>")
+     *   .addTo(map);
+     * var popupElem = popup.getElement();
+     * popupElem.style.fontSize = "25px";
```

Comment:
moving over this comment from @katydecorah: 

---

(Please let me know if you think of a better example. I looked at the original request for this feature #7425 and saw that the user wanted to dynamically change CSS classes, however, Popup already has class-related utilities.)

Reply:
Do you have an opinion on this one @danswick?

Reply:
i think it's fine! let's ship it and see if we get any feedback :mischief_giggle: